File: Algoritmos Traduzidos/Perl/Algoritmos_Filipe_Farinha/10.pl
#Programa que imprime a soma de todos os numeros inteiros num intervalo introduzido pelo utilizador.
print "Limite inferior:\n";
$li = <STDIN>;
print "Limite superior:\n";
$ls = <STDIN>;
while ($li <= $ls)
{
	$s = $s + $li;
	$li = $li + 1;
}
print "$s\n";

File: Algoritmos Traduzidos/Ruby/Algoritmos_Filipe_Farinha/10.rb
#Programa que imprime a soma de todos os numeros inteiros num intervalo introduzido pelo utilizador.
puts "Limite inferior:"
li = gets
puts "Limite superior:"
ls = gets
while li.to_i <= ls.to_i
  s = s.to_i + li.to_i
  li = li.to_i + 1
end
  puts "#{s}\n"